src/Elcodi/Store/PaymentBridgeBundle/Services/PaymentBridge.php

```diff
@@ -172,9 +172,13 @@ public function getExtraData()
                     ->getPurchasable()
                     ->getName();
 
+                /*
+                 * Elcodi stores amounts in integer cents (10USD = 1000)
+                 * but Paypal expects decimal (10.00USD)
+                 */
                 $orderLineArray['amount'] = $orderLine
                     ->getAmount()
-                    ->getAmount();
+                    ->getAmount() / 100;
```

Comment:
@alch PaymentSuite should have an specification. It means that the bridge should always pass the money in a specific format (right now, no-comma format). A bridge should be always the same from every kind of payment implementation, and given this bridge specification (1000), every implementation should take care about internal payment requirements.

As you can see here http://payment-suite.readthedocs.org/en/latest/configuration.html#paymentbridge-service, the getAmount() always return the amount in cents, so Paypal Bundle must work with that API.